README.md

```diff
@@ -0,0 +1,45 @@
+# Discovery based on Solid Social Agent WebId
+
+## Overview
+
+When an app allows a user to log in to a Solid Identity Provider, the app receives back an authenticated identifier for the user - their WebId.  The general purpose of this document is to describe what an app can discover based on the WebId.  More specific purposes are described below.
```

Comment:
"authenticated identifier" is unclear. I would use "verifiable token identifiying a user". For instance, OpenID connect will return a signed ID token. OAuth2 will return a signed bearer token, a DID-Auth based mechanism will return something different, etc. The point is that there is a mechanism for verifying that this data hasn't been tampered with.

Reply:
If this document is building on the [draft WebID specification](https://www.w3.org/2005/Incubator/webid/spec/identity/), that should be made explicit. And if so, the canonical spelling of the term is `WebID`.

Reply:
Will change WebId -> WebID, thanks.  Added WebID spec to related resources.  

In regard to "the app receives back an authenticated identifier" - I really want to focus on the WebID, not the token, would this work for you?

     the app receives back a verifiable identifying token and a WebId

Reply:
Right, the format of the response is beyond the scope of this document. The relevant point is that the data is trusted and that it establishes the WebID of the user. That trust can be established via cryptographic signatures or other forms.

I might prefer something more along the lines of "the Solid identity provider supplies the app with data that establishes the identity of the user in terms of a WebID URI". You might even add something about how "the format of this data and the mechanism by which this data is retrieved and verified is beyond the scope of this document". That would leave the door open to OpenID, SAML, GNAP, etc.

Reply:
Okay, I went with your suggested phrasing, thanks.